jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,891 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+Nim version 1.6 is now officially released!
+
+A year in the making, 1.6 is the latest stable release and by far the largest yet.
+We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in more than 40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and in the last 2 months the top-50 ([https://forum.nim-lang.org/t/8297](https://forum.nim-lang.org/t/8297)).
+We hope this release will reinforce this trend, building on Nim's core strengths:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; Lisp-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
+
+Why use Nim?
+* one language to rule them all: write your iOS/Android/web frontend and backend in the same language, from a [Gameboy kickstarter](https://forum.nim-lang.org/t/8375) to a [blockchain client](https://github.com/status-im).
```

Comment:
This sentence is kind of awkward, our Kickstarter isn't an example of the same language being used for frontend and backend. The project is already mentioned in another bullet point so you don't have to mention it here (also in general I'd prefer not to be seen right next to blockchain stuff) :sweat_smile:

Reply:
sure; so we need to come up with something else in its place; suggestions from the community welcome

Reply:
one language to rule them all: Good for everything, from [shell scripting](https://nim-lang.org/docs/nims.html) to [web front and backend](https://github.com/nim-lang/nimforum), to [scientific computing](https://github.com/SciNim), [blockchain client](https://github.com/status-im), [gamedev](https://github.com/ftsf/nico), and [embedded](https://github.com/EmbeddedNim).

Reply:
thanks, good, i edited this a bit and added `[deep learning](https://github.com/mratsim/Arraymancer)`

this line is important, we should polish it and find salient examples, further ideas / edits welcome

Reply:
There are two games can be installed on android and iOS
https://www.frayhem.com/en/ (based on godot-nim)
https://github.com/pragmagic/godot-nim#made-with-godot-nim
https://oftengames.com/ (should be based on nimx or [rod](https://github.com/yglukhov/rod) I'm not sure.

Reply:
see last few edits, how does it look now?